Driverless AI/machine-learning-experiment-scoring-and-analysis-financial-focus/machine-learning-experiment-scoring-and-analysis-financial-focus.md

```diff
@@ -831,65 +574,65 @@ Where:
+When plotting lift, we also plot it against quantiles to visualize how likely it is that a positive case will occur since the Lift chart is derived from the cumulative gains chart. The lift curve points are calculated by determining the ratio between the result predicted by our model and the result using a random model (or no model). For instance, assuming a base rate (or hypothetical threshold) of 20% from a random model, we would take the cumulative gain percent at the 20% quantile, X, and divide it by 20. We do this for all the quantiles until we get the full lift curve. 
 
-We can start the lift chart with the base rate as seen below, recall that the base rate is the target threshold.
+We can start the lift chart with the base rate as seen below; recall that the base rate is the target threshold.
 
-![lift-chart-base-rate](assets/lift-chart-base-rate.jpg)
+![lift-chart-base-rate](assets/lift-chart-base-rate.png)
 
-When looking at the cumulative lift for the top quantiles, X, what it means is that when we select lets say 20% from the quantile from the total test cases based on the mode, we can expect X/20 times the total of the number of positive cases found by randomly selecting 20% from the random model.
+When looking at the cumulative lift for the top quantiles, X, means is that when we select, let's say 20% from the quantile from the total test cases based on the mode, we can expect X/20 times the total of the number of positive cases found by randomly selecting 20% from the random model.
 
 
-![lift-chart](assets/lift-chart.jpg)
+![lift-chart](assets/lift-chart.png)
 
 ### Kolmogorov-Smirnov Chart 
 
-Kolmogorov-Smirnov or K-S measures the performance of classification models by measuring the degree of separation between positives and negatives for validation or test data [13]. 
+Kolmogorov-Smirnov or K-S measures the classification models' performance by measuring the degree of separation between positives and negatives for validation or test data [13]. 
 
-The KS statistic is the maximum difference between the cumulative percentage of responders or 1's (cumulative true positive rate) and cumulative percentage of non-responders or 0's (cumulative false positive rate). The significance of KS statistic is, it helps to understand what portion of the population should be targeted to get the highest response rate (1's) [17].
+The KS statistic is the maximum difference between the cumulative percentage of responders or 1's (cumulative true positive rate) and cumulative percentage of non-responders or 0's (cumulative false positive rate). The KS statistic's significance is that it helps to understand what portion of the population should be targeted to get the highest response rate (1's) [17].
 
-![k-s-chart](assets/k-s-chart.jpg)
+![k-s-chart](assets/k-s-chart.png)
 
```

Comment:
Everything above in this section is good content though I think we need to add a quick overview and layout at the start of the task and have the layout guide us when explaining different concepts...I feel that transitions are missing when moving to other concepts...for example, we can say: we are going to talk about x and now y because is an extension of x etc.